wtsapi/syscall.go

```diff
@@ -111,7 +111,7 @@ func EnumerateSessions(server syscall.Handle) (sessions []SessionEntry, err erro
 
 	// Cast the pointer to an unbounded array and then take a slice of
 	// suitable size from it
-	list := ((*[math.MaxInt]sessionEntry)(ptr))[0:count:count]
+	list := ((*[math.MaxInt/unsafe.Sizeof(sessionEntry{})]sessionEntry)(ptr))[0:count:count]
```

Comment:
I'm not sure this is necessary because the array size here is an arbitrarily large value. Did this solve an issue that you encountered?

Reply:
If I try to build this library with GOOS=Windows GOARCH=386 (host machine is Windows-amd64) the compilation fails. Unless I apply this patch.

I guess it makes sense, because pointers to sessionEntries at the end of the array would not fit in 32-bit memory.